x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/logs_ui_categories/ml/log_entry_categories_count.json

```diff
@@ -0,0 +1,34 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "Categorize log entries and detect count anomalies",
+  "groups": ["logs-ui"],
+  "analysis_config": {
+    "bucket_span": "15m",
+    "categorization_field_name": "message",
+    "detectors": [
+      {
+        "by_field_name": "mlcategory",
+        "detector_description": "count by mlcategory",
```

Comment:
Or even 'count by log entry category'?

Reply:
sounds good, this was just auto-generated by the ML setup wizard :wink: